pkg/bootstrap/api/types.go

```diff
@@ -46,6 +46,13 @@ const (
 	// other value is assumed to be false. Optional.
 	BootstrapTokenUsageSigningKey = "usage-bootstrap-signing"
 
+	// BootstrapTokenUsageAuthentication signals that this token should be used
+	// as a bearer token to authenticate against the Kubernetes API. The bearer
+	// token takes the form "(token-id):(token-secret)" and authenticates as the
```

Comment:
The form should be `<token-id>.<token-secret>`.  

We've been teaching users that this is what a kubeadm bootstrap token looks like and we should remain consistency.